flink-state-backends/flink-statebackend-rocksdb/src/main/java/org/apache/flink/contrib/streaming/state/RocksDBKeyedStateBackend.java

```diff
@@ -658,8 +685,6 @@ public void notifyCheckpointAborted(long checkpointId) throws Exception {
                         stateDesc, snapshotTransformFactory, newMetaInfo.getStateSerializer());
         newMetaInfo.updateSnapshotTransformFactory(wrappedSnapshotTransformFactory);
 
-        ttlCompactFiltersManager.configCompactFilter(stateDesc, newMetaInfo.getStateSerializer());
```

Comment:
Could you explain why ths call was removed?

Reply:
Actually, it's just the only place different between two `createInternalState()`.
compactFilter cannot be registered twice currently, so I just make `createInternalState` who means creating state but not which means creating or updating call it.

Reply:
Thanks for the explanation.